bpf/aya-bpf-macros/src/lib.rs

```diff
@@ -344,3 +345,84 @@ pub fn socket_filter(attrs: TokenStream, item: TokenStream) -> TokenStream {
         .unwrap_or_else(|err| err.to_compile_error())
         .into()
 }
+
+/// Marks a function as a fentry eBPF program that can be attached to almost
+/// any function inside the kernel. The difference between fentry and kprobe
```

Comment:
nit: That's only one use case. You can also attach fentry/fexit to other eBPF programs

Reply:
Added a sentence about that.